content/projects/bootcamp-writing-assignment/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: Bootcamp Writing Assignment
+---
+
+### We would like to get to know you better.
+
+In this assignment you are going to create a .md file with a 1000 word essay on yourself, in this essay please can you try cover the below points:
+
+ - Your personal history, where you come from, feel free to include some interesting facts about yourself.
+
+ - Why you would like to become a coder.
+
+ - Why you decided on this particular department at Umuzi.
+
+ - Your past extracurricular activities or work experiences.
```

Comment:
A fun hack would be to add these questions: (feel free to reword)

- Why do you think you will succeed here?

- The courses offered by Umuzi are hard - a lot of people struggle for lots of different reasons. It's our job to help you succeed so it's good for us to know what to expect so we can do a good job. What weaknesses do you have that might stand in the way of your success? And what stratergies do you have for overcoming these challenges?

- If you succeed at Umuzi and get a high value career as a developer or data scientist, what do you think your life will look like? Do you have personal goals? How will this effect your family and your community? Anything else?